content/docs/command-reference/dag.md

```diff
@@ -7,43 +7,25 @@ specified stage. By default it lists
 ## Synopsis
 
 ```usage
-usage: dvc pipeline show [-h] [-q | -v] [-c | -o] [-l] [--ascii]
-                         [--dot] [--tree]
-                         [targets [targets ...]]
+usage: dvc dag [-h] [-q | -v] [--dot] [--full] [target]
 
 positional arguments:
-  targets         DVC-files to show pipeline for. Optional.
-                  (Finds all DVC-files in the workspace by default.)
+  targets         Stage or output to show pipeline for.
+                  Optional. (Finds all stages in the workspace by default.)
 ```
 
 ## Description
 
-`dvc show` displays the stages of a pipeline up to one or more target DVC-files
-(stage files). If specific `targets` are omitted, `Dvcfile` will be assumed. The
-`-c` and `-o` options allow to list the corresponding commands or data file flow
-instead of stages.
-
-> Note that the stages in these lists are in descending order, that is, from
-> first to last.
+`dvc dag` displays the stages of a pipeline up to the target stage. If `target`
+is omitted, it will show the full project DAG.
```

Comment:
- Is it possible to have more than one DAG?
- Should we call it always "pipeline" or always "DAG" or always "dependency graph"? We currently have a mix of all these everywhere 😕

Reply:
Yes, 2 DAGs together is 1 DAG too, just with weakly connected components :)

Pipeline is a DAG in which all stages are somehow connected with each other, so it is not quite that. We could call it pipelineS, but `dvc add` foo is not quite a pipeline strictly speaking. `dependency graph` sounds as the most correct term, but DAG is a synonym, hence the https://dagshub.com/ :) So we could and probably should use them interchangeably.

This doc is adapted from the old `dvc pipeline show` so it does suffer from some legacy sentence structure :(